programmers2/수박수박수박수박수.js

```diff
@@ -0,0 +1,8 @@
+function solution(n) {
+    let suBak = "" ;
+    for (let i = 1; i <= n; i++) 
+        (i % 2) ? suBak += "��" : suBak += "��";
```

Comment:
true와 flase 에서 subak 을 각각 부르는것보단
아래와 같이 한번에 하는것이 더욱 좋다.
질문 생기면 카톡으로 연락하시면 됩니다~~ 
```suggestion
         suBak += i % 2 ?  "¼ö" : "¹Ú";
```